soccer/gameplay/evaluation/double_touch.py

```diff
@@ -39,7 +39,7 @@ def __init__(self):
         self.add_transition(
             DoubleTouchTracker.State.restart_play_began,
             DoubleTouchTracker.State.kicking,
-            lambda: any(bot.has_ball() for bot in main.our_robots()),
+            lambda: (any(bot.has_ball() for bot in main.our_robots())) and not main.game_state().is_placement,
```

Comment:
The `is_placement` needs parentheses since it's a method rather than a property.  Should be `is_placement()`.